WordPress/pattern-directory
# Theme: Make Menu & Search components more universal

The `CategoryMenu` & `CategorySearch` components don't need to be tied to "category", since they both accept options that can display different content. This PR switches them to `Menu` and `Search`. `Menu` will (in another PR) be used on the My Patterns and Favorites pages, see #16.

### Screenshots

No visual changes.

### How to test the changes in this Pull Request:

1. View any grid pages
2. The category navigation and search should work as expected
